inc/Classes/DB.php

```diff
@@ -377,4 +386,13 @@ public function DisplayErrors()
             $this->errors = '';
         }
     }
+
+    /**
+     * Sets the SQL Mode for this database session.
+     */
+    public function setSqlMode(string $sqlmode)
+    {
+        $sqlModeQuery = sprintf("SET SESSION SQL_MODE='%s';", mysqli_real_escape_string($this->link_id, $sqlmode));
```

Comment:
As per prior comment: escaping is not really needed here, but it doesn't hurt, fine to leave as-is